docs/rabbitmq-server.service.example

```diff
@@ -10,6 +10,14 @@ User=rabbitmq
 Group=rabbitmq
 NotifyAccess=all
 TimeoutStartSec=3600
+# Note:
+# You *may* wish to add the following to automatically restart RabbitMQ
+# in the event of a failure. This is not recommended as a replacement
```

Comment:
Some may read this as “this is not recommended”. How about “systemd service restarts are not a replacement for service monitoring”?